### ISSUE TITLE
report labels the default 10,000 steps as "Full training run." — the top bucket starts at the baseline

### ISSUE THREAD

**Issue by ACEFGI** [inanimate-tech/wakeword-forge#4]
## Summary

`training_depth_phrase()` labels any run of 10,000 steps or more as "Full training run." — but 10,000 is the default. The ceiling label starts exactly at the baseline, so the report cannot distinguish a default run from a genuinely long one, and tells the user their baseline run was "full".

## Detail

`report.py:56-61`:

```python
def training_depth_phrase(steps: int) -> str:
    if steps < 2000:
        ...
    if steps < 10000:
        ...
    return "Full training run."
```

With `--steps` defaulting to 10,000 (`cli.py:20`), every ordinary run lands in the top bucket. A 40,000-step run gets the same words as a run four times shorter.

## Why 10,000 is not "full"

The pipeline's defaults are a parameter-for-parameter copy of upstream's [basic training notebook](https://github.com/kahrendt/microWakeWord/blob/main/notebooks/basic_training_notebook.ipynb) — `training_steps [10000]`, `learning_rates [0.001]`, `batch_size 128`, class weights `[1]`/`[20]`, all spec-augment masks `[0]`, `eval_step_interval 500`, `clip_duration_ms 1500`, 1,000 TTS samples. That is the reference *baseline*.

For contrast, [esphome/micro-wake-word-models#20](https://github.com/esphome/micro-wake-word-models/pull/20) reports community "advanced" runs at 35,000 samples / 15,000 steps and 50,000 samples / 40,000 steps — roughly 16 hours on CPU each. (Second-hand from the PR discussion; there is no advanced notebook in either upstream repo, only `basic_training_notebook.ipynb`.)

Measured here: a default run is ~9m15s end to end on an M5 (generate ~1m, features 1m13s, prebake 2m13s, **train 5m08s**, export 9s). Nothing is being skipped — the model is a 60 KB 4-block mixednet, so ~33 steps/sec at batch 128 is simply what a model this small does. But "Full training run." invites the reader to conclude they have exhausted the available depth, when they have used the starting point.

## Suggested direction

Re-scale the buckets so the top one means something, e.g. treat the default as "baseline" and reserve the strongest wording for multiples of it:

- `< 2000` — quick test
- `< 10000` — short run
- `10000` — baseline (upstream's reference settings)
- `> 10000` — extended run
- `>= 40000` — long run

Naming the default as the default is the important part; the exact thresholds matter less.

Related: #1 (accent/speaker coverage), where `--samples` matters more than `--steps`.
